### PR TITLE
[SUCE-377] donations data clean up

### PR DESCRIPTION
**Pull request by SamaneriVimala**

### CODE REVIEW

File: client/elements/sc-donate-now-page.html [hunk 1]
```diff
@@ -485,6 +484,16 @@
...
           }
 
+          resetForm() {
+              this.$.donation_form.reset();
+              this.shadowRoot.querySelector('.amount-input').value = undefined;
+              this.shadowRoot.querySelector('.one-time-donation').checked = true;
+              this.shadowRoot.querySelector('.monthly-donation').checked = false;
+              this.shadowRoot.querySelector('.name-input').value = undefined;
+              this.shadowRoot.querySelector('.email-input').value = undefined;
+              this.shadowRoot.querySelector('.message-input').value = undefined;
```
Comment (TrebuhD) [inline client/elements/sc-donate-now-page.html]: Are all these values not set by calling `this.$.donation_form.reset();`?

File: client/elements/sc-page-static.html [hunk 2]
```diff
@@ -283,6 +283,7 @@ <h3>{{localize('pageNotFound')}}</h3>
       }
 
       _changeView() {
+          this.$.donate.resetForm();
```
Comment (TrebuhD) [inline client/elements/sc-page-static.html]: Will this get called when going from a static page to a non-static page (the suttaplex for example)?

If it doesn't get called, we can also put this function call it a `disconnectedCallback()`.

File: client/elements/sc-donate-now-page.html (hunk 1 above)
Comment (SamaneriVimala) [reply client/elements/sc-donate-now-page.html]: No, that only resets the fields for the amount and the credit card details but not the rest.

File: client/elements/sc-page-static.html (hunk 2 above)
Comment (SamaneriVimala) [reply client/elements/sc-page-static.html]: I tested it and it works fine going from the static to the non-static page. I think that was never a problem in the first place anyway.

**Pull request merged**